src/Glpi/Controller/GenericAjaxCrudController.php

```diff
@@ -36,29 +36,29 @@
 namespace Glpi\Controller;
 
 use CommonDBTM;
-use Glpi\Http\Response;
+use Glpi\Form\Form;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
 
 /**
  * Ajax controller for handling "update", "delete", "restore" and "purge"
  * actions on CommonDBTM objects
  */
-class CommonAjaxController
+class GenericAjaxCrudController extends AbstractController
 {
     /**
      * Object to update, delete, restore or purge
      * @var CommonDBTM
      */
     protected CommonDBTM $item;
 
-    /**
-     * Handle a given HTTP request
-     *
-     * @param array $input request data
-     *
-     * @return Response
-     */
-    final public function handleRequest(array $input): Response
+    #[Route("/generic-ajax-crud", name: "glpi_ajax_generic-crud")]
```

Comment:
This route seems weird since you insisted on using pascal case everywhere else.
Should be `/GenericAjaxCrud` and `glpi_ajax_generic_crud`.

Reply:
Fixed during the rebase.